index.js

```diff
@@ -23,8 +23,8 @@ const get_remainder = x => {
   return x.substr(index + 1);
 };
 
-const Level = (level_code, submitter) => {
-  return { code: level_code, submitter: submitter };
+const Level = (level_code, submitter, username) => {
```

Comment:
While this will certainly work, I think the long term solution is to use username for everything behind the scenes, and display name only when interacting with chat.  I've toyed with building an in memory map of the chatters so that we can easily go between the username and the display name.  For example, storing the display name can come back to haunt us as display names can be changed in the middle of the stream. Having the lookup table that is modified when people chat helps to mitigate this.  However this change should be good for now.

Reply:
That feature would be very great as well as also allowing the username as arguments for commands and also allowing the @ symbol in front of usernames and display names.